src/charm.py

```diff
@@ -431,25 +443,40 @@ def k8s_auth(self):
 
         self._authed = True
 
+    def k8s_api(self, api: str) -> object:
```

Comment:
I'm curios what the reason was for adding this?

Reply:
Removed :)